### ISSUE TITLE
Timeline | change axis location

### ISSUE THREAD

**Issue by Shaboo** [almende/vis#3676]
how to change the top axis to become above the bottom axis?

**Comment by brichbe** [comment]
Do you mean the [Timeline Option](http://visjs.org/docs/timeline/#Configuration_Options) `orientation.axis` ?  Try setting that to `'top'` or `'both'`.

**Comment by Shaboo** [comment]
@brichbe 

No, I want the top axis to be rendered exactly above the bottom axis

**Comment by brichbe** [comment]
No, I don't believe that's possible, and it's difficult to understand why you'd want that.  Can you supply a picture to show what you want it to look like?
